web/src/components/FilingsCalendar.tsx

```diff
@@ -125,7 +126,13 @@ export const FilingsCalendar = (props: Props): ReactElement => {
             return (
               <tr key={rowIndex}>
                 {monthIndicesForRow.map((month) => {
-                  return (
+                  const date = getJanOfCurrentYear().add(month, "months");
```

Comment:
I don't think this line of code does anything. You're using the variable `month` to create a new variable `date`, so that you can make use of `date.month()` on the next line.

Right after this line, do a console.log of `month` and of `date.month()` and you should see what I mean